pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+ baseurl = "https://redwax.eu/dist/rs/";
+ name = "mod_ca";
+ suffix = ".tar.gz";
+
+ meta = with stdenv.lib; {
+   description = "RedWax CA service modules.";
```

Comment:
No period at the end of `description` please.

Reply:
fixed everywhere.